src/plugins/embeddable/public/lib/actions/edit_panel_action.ts

```diff
@@ -94,6 +94,7 @@ export class EditPanelAction implements Action<ActionContext> {
       const oldExplicitInput = embeddable.getExplicitInput();
       const newExplicitInput = await factory.getExplicitInput(oldExplicitInput, embeddable.parent);
       embeddable.parent?.replaceEmbeddable(embeddable.id, newExplicitInput);
+
```

Comment:
**nit:** Oops - this extra line here is probably from my PR 😅 Probably best to remove it hehe